src/Mod/PartDesign/App/FeaturePipe.h

```diff
@@ -88,4 +88,4 @@ class PartDesignExport SubtractivePipe : public Pipe {
 } //namespace PartDesign
 
 
-#endif // PART_Pad_H
+#endif // PART_Pipe_H

```

Comment:
It specifies the wrong module too, s/part/partdesign/
```suggestion
#endif // PARTDESIGN_Pipe_H
```